site/content/contribute/getting-started/guilds.md

```diff
@@ -0,0 +1,50 @@
+---
+title: "Engineering Guilds"
+date: 2021-01-25T16:50:35+00:00
+weight: 50
+subsection: Getting Started
+---
+
+Our Engineering Guilds are a forum for sharing knowledge, discussing and agreeing on plans, and disseminating information related to a particular subject or technical topic. Many (although not all) Engineering Guilds are affiliated to a particular "platform" that forms a part of Mattermost, and are therefore associated with a particular Platform Team. However, the participants in each Guild span many engineering teams at Mattermost in order to get people together to share knowledge and disseminate decisions across the Engineering organization.
+
+The main activity of Guilds is a regular meeting. These meetings are public, and anyone within Mattermost and the wider community is welcome to attend. Details are provided below on each of the Guilds, including how to participate.
+
+## Web Application & Desktop Guild
+
+The Web Application & Desktop Guild covers all areas related to front-end web development and Electron desktop app development at Mattermost.
+
+- **Guild Lead:** Dean Whillier
+- **Guild Channel:** [~Developers: WebApp](https://community.mattermost.com/core/channels/webapp)
+- **Guild Meeting:** *(see header of Guild Channel for meeting invite)*
```

Comment:
Should that be 
```suggestion
- **Guild Meeting Time:** *(see header of Guild Channel for meeting invite)*
```
? It wasn't obvious to me what that means.

Reply:
0/5 personally. I'll let @justinegeffen advise as editor.

Reply:
Good suggestion, @hanzei.